.mergify.yml

```
@@ -0,0 +1,21 @@
+pull_request_rules:
+  - name: automatic merge on CI success and review
+    conditions:
+      - status-success=continuous-integration/travis-ci/pr
+      - "#approved-reviews-by>=1"
+      - "#review-requested=0"
+      - "#changes-requested-reviews-by=0"
+      - base=master
+      - label=['ready-to-merge']
+    actions:
+      delete_head_branch: True
```

```suggestion
      delete_head_branch: {}
```
Not sure if this matters